### PR TITLE
document schemes list and schemes/duty API endpoints

### PR DESCRIPTION
**Pull request by ansonK**
Initial rundown of internal API that the Rota App will expose

Uses API blueprint spec:

http://apiary.io/blueprint
https://github.com/apiaryio/api-blueprint/blob/master/API%20Blueprint%20Specification.md

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            "errors": [
+                "You must provide a valid Authorization Token"
+            ]
+        }
+
+## Duty Solicitor [/api/v1/schemes/{id}/duty?{at_time}]
+Returns the Profile of the Solicitor that is on Duty for the given scheme at the current time
+
++ Parameters
+    + at_time (unix timestamp)
```
Comment (pixeltrix) [inline apiary.apib]: Should this be a ISO 8601 timestamp?

Comment (pixeltrix) [inline apiary.apib]: Should this be a ISO 8601 timestamp?

File: apiary.apib [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
++ Response 200 (application/json)
+
+        {
+            "schemes_url": "/api/v1/schemes"
+        }
+
+# Group Schemes
+Schemes contain a List of Solicitors that are on Duty
+
+## Schemes Collection [/api/v1/schemes]
```
Comment (pixeltrix) [inline apiary.apib]: Do we need to be able to filter this list? How many schemes are there across the country?

**Comment by pixeltrix** [comment]
One other question - is this an internal API only? May there be an need for external services to call it? Maybe as part of the government Open Data program?

**Comment by pedrosmmoreira** [comment]
@pixeltrix, @ansonK With the exception of the question regarding the timestamp formatting, I'm happy to get this merged in order to move forward with a stable example payload.

**Comment by ansonK** [comment]
@pedrosmmoreira timestamps are now ISO 8061

**Comment by pedrosmmoreira** [comment]
Thanks, merging.

**Pull request merged**